config/envs/default.py

```diff
@@ -127,3 +133,60 @@ class DefaultConfig:
     }
 
     USE_LOCAL_DATA = strtobool(getenv("USE_LOCAL_DATA", "False"))
+
+    COF_FUND_ID = "47aef2f5-3fcb-4d45-acb5-f0152b5f03c4"
+    COF_R2_ROUND_ID = "c603d114-5364-4474-a0c4-c41cbf4d3bbd"
```

Comment:
Doesn't seem like these are being used at all?

Reply:
true, ill remove. We need them to get fund info from the fund store (https://github.com/communitiesuk/funding-service-design-frontend/blob/fs-1288-refactor-with-new-application-store/app/models/application.py#:~:text=%5D%20%3D%20form-,%23%20TODO%20post%20mvp,-%23%20Update%20the%20section) 
...however the fund_id and round_id are available on the application object 👍